morph_tool/converter.py

```diff
@@ -1,8 +1,11 @@
 """A morphology converter that tries to keep the soma surface equal."""
 import logging
 from pathlib import Path
+import os
+import uuid
 
 import numpy as np
+from scipy.optimize import minimize_scalar
```

Comment:
Scipy is missing in the requirements, it should be added even though it's already an indirect req.